src/PKSim.Core/Mappers/ParameterListOfValuesRetriever.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Linq;
+using OSPSuite.Core.Domain;
+using OSPSuite.Utility.Collections;
+using PKSim.Assets;
+using PKSim.Core.Model;
+
+namespace PKSim.Core.Mappers
+{
+   public interface IParameterListOfValuesRetriever
```

Comment:
Move this out of the DTO mapper into its own file so that I can use it here as well